README.adoc

```diff
@@ -98,61 +98,41 @@ export DOCKER_API_VERSION=1.20
 the time of `vagrant up` itself. Every run of
 `vagrant service-manager env docker` checks for the validity of the
 certificates on the host machine by matching the certificates inside the
-box. If the certificates on the host machine are invalid, this command
+VM. If the certificates on the host machine are invalid, this command
```

Comment:
I find this too vague. The VM is more than a "development environment" imo. So far I find VM the most fitting term